generators/server/templates/src/main/java/package/service/_UserService.java

```diff
@@ -9,9 +9,19 @@
 import <%=packageName%>.security.AuthoritiesConstants;
 import <%=packageName%>.security.SecurityUtils;
 import <%=packageName%>.service.util.RandomUtil;
+<% if (databaseType == 'sql' || databaseType == 'mongodb') { %>
+import <%=packageName%>.web.rest.util.PaginationUtil;
+<%_ } _%>
 import <%=packageName%>.web.rest.vm.ManagedUserVM;
```

Comment:
OK, so first I think we should refactor this part, so the service layer doesn't call the web layer, before we can update this part